src/YQWizard.cc

```diff
@@ -1097,6 +1106,49 @@ void YQWizard::showHelp()
 }
 
 
+void YQWizard::showHotkeys()
+{
+    /**
+     * Help text to be shown after pressing Shift-F1 listing the advanced
+     * keyboard shortcuts available in the Qt-UI
+     **/
+    _qHotkeysText = _(
+        "<h1>Advanced Hotkeys</h1>"
+        "<dl>"
+        "<dt>Ctrl-Alt-Shift-X</dt>"
+        "<dd>open a terminal window (xterm), useful for VNC installs</dd>"
+        "<dt>Shift-F7</dt>"
+        "<dd>Debug level</dd>"
+        "<dt>Ctrl-Shift-D</dt>"
+        "<dd>Debug module write</dd>"
+        "<dt>Print key</dt>"
+        "<dd>Screenshot</dd>"
+        "<dt>Shift-F8</dt>"
+        "<dd>Save logs to</dd>"
+        "<dt>Shift-F4</dt>"
+        "<dd>Accesibility, Switch to vision impaired palette</dd>"
+        "<dt>Ctrl-Shift-Alt-M</dt>"
+        "<dd>Start/Stop macro recorder</dd>"
+        "<dt>Ctrl-Shift-Alt-P</dt>"
+        "<dd>Play macro</dd>"
+        "<dt>Ctrl-Shift-Alt-T</dt>"
+        "<dd>Dump widget tree to y2log</dd>"
```

Comment:
Speaking about all the comments about the help text I used: It is a direct copy&paste from
https://en.opensuse.org/SDB:YaST_tricks#YaST_Hotkeys
So let's fix it there first - then I can take it back to here.